src/css/custom.css

```
@@ -6,113 +6,98 @@
 }
 
-@font-face {
-  font-family: "General Sans";
-  src: url(https://gruntwork.io/_next/static/media/a34f9d1faa5f3315-s.p.woff2);
-  font-weight: 700;
-  font-style: normal;
-  font-display: swap;
-}
-
 @font-face {
   font-family: "Inter";
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)